Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (0.9.2)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
     mini_mime (1.1.5)
     mini_portile2 (2.8.5)
     minitest (5.11.3)
-    minitest-rails (6.1.0)
+    minitest-rails (6.1.1)
       minitest (~> 5.10)
       railties (~> 6.1.0)
     mixlib-shellout (3.2.7)
       chef-utils
-    mocha (1.2.1)
-      metaclass (~> 0.0.1)
+    mocha (1.16.1)
     momentjs-rails (2.29.4.1)
       railties (>= 3.1)
-    msgpack (1.4.2)
+    msgpack (1.7.2)
     multi_json (1.15.0)
     multi_xml (0.6.0)
     multipart-post (2.3.0)
-    mysql2 (0.5.3)
+    mysql2 (0.5.5)
     net-http-persistent (4.0.2)
       connection_pool (~> 2.2)
-    net-ldap (0.18.0)
+    net-ldap (0.19.0)
     netrc (0.11.0)
-    newrelic_rpm (6.7.0.359)
+    newrelic_rpm (6.15.0)
```

Comment:
### [newrelic_rpm](https://newrelic.com/ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 6.7.0.359 → 6.15.0

[_[change-log](https://github.com/newrelic/newrelic-ruby-agent/blob/main/CHANGELOG.md), [source-code](https://github.com/newrelic/newrelic-ruby-agent), [gem-diff](https://my.diffend.io/gems/newrelic_rpm/6.7.0.359/6.15.0)_]





<details>
<summary>Commits</summary>

A change of **1520** commits. See the full changes on [the compare page](https://github.com/newrelic/newrelic-ruby-agent/compare/6.7.0.359...6.15.0).

These are the first 10 commits:
- (efe815f) [try that cache](https://github.com/newrelic/newrelic-ruby-agent/commit/efe815fa2e31b71b831cf9149236d933a31f3a84)
- (94af79f) [always set up the environment](https://github.com/newrelic/newrelic-ruby-agent/commit/94af79f1d4dc4ee5d7ab981608316606a26c362c)
- (c5f1f41) [it is restored if it's cached.  use power](https://github.com/newrelic/newrelic-ruby-agent/commit/c5f1f415dfaf0e060824af362cfce5e5be5294b3)
- (8d01ef9) [trying cache again](https://github.com/newrelic/newrelic-ruby-agent/commit/8d01ef91ced154d6197caa1754caa99a5f747a0d)
- (cc57d58) [env.HOME in flight](https://github.com/newrelic/newrelic-ruby-agent/commit/cc57d58a5aa1b0ae2d22c11bcd413550b31ed0d9)
- (099e5fc) [back to tilde](https://github.com/newrelic/newrelic-ruby-agent/commit/099e5fcdc6734571edd56ba8a27469b53fe2d592)
- (e1e4565) [tell us we're skipping town](https://github.com/newrelic/newrelic-ruby-agent/commit/e1e45657199046d9ddc06679932e80d4c4127404)
- (7f477b0) [removing extraneous files](https://github.com/newrelic/newrelic-ruby-agent/commit/7f477b0aee67bdf89e11697e36ad8d326ccb1cef)
- (c6d914b) [Merge remote-tracking branch 'upstream/main' into main](https://github.com/newrelic/newrelic-ruby-agent/commit/c6d914b808493e67fdd6a34e00e2bbefb460bab8)
- (49fe7a0) [Merge branch 'main' into build-ruby-action](https://github.com/newrelic/newrelic-ruby-agent/commit/49fe7a0271647aa8ac882a9a414cd8777d386be5)


</details>